docs/UserGuide.md

```diff
@@ -189,20 +190,40 @@ Format: `delClient i/INDEX`
 Examples:
 * `delClient i/2`
 
+#### 3.1.5 Editing a client: `editClient`
+
+Edits details of the specified client.
+
+Format: `editClient i/INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
+
+* Edits the client at the specified `INDEX`.
```

Comment:
Should probably specify where the user can get the `INDEX` which in this case is from `listClient` command.